docs/download.md

```diff
@@ -43,7 +43,7 @@ for your hardware's minimum driver version.
 ### Download v21.12.0
```

Comment:
```suggestion
### Download v21.12.2
```

Reply:
Since rapids is v21.12.0, so should we follow the previous format as v21.08? 
![image](https://user-images.githubusercontent.com/84758614/146730415-954e372a-9c82-46e0-9c49-152e74dcc86b.png)